common/src/main/java/org/apache/falcon/metadata/MetadataMappingService.java

```diff
@@ -275,64 +303,49 @@ public Void execute(TransactionalGraph transactionalGraph) throws Exception {
         }
     }
 
-    private void onSuccessfulExecution(final WorkflowExecutionContext context) throws FalconException {
+    private void updateInstanceStatus(final WorkflowExecutionContext context) throws FalconException {
+        if (context.getContextType() == WorkflowExecutionContext.Type.COORDINATOR_ACTION) {
+            // TODO(yzheng): FALCON-1776 Instance update on titan DB based on JMS notifications on coordinator actions
+            return;
+        }
+
         WorkflowExecutionContext.EntityOperations entityOperation = context.getOperation();
         switch (entityOperation) {
         case GENERATE:
-            onProcessInstanceExecuted(context);
+            updateProcessInstance(context);
             break;
         case REPLICATE:
-            onFeedInstanceReplicated(context);
+            updateReplicatedFeedInstance(context);
             break;
         case DELETE:
-            onFeedInstanceEvicted(context);
+            updateEvictedFeedInstance(context);
             break;
         case IMPORT:
-            onFeedInstanceImported(context);
+            updateImportedFeedInstance(context);
             break;
```

Comment:
EXPORT operation is missing originally. We need to address that separately via JIRA.